### PR TITLE
Can I buy or switch account to one of the old plans?

### PR DESCRIPTION
**Pull request by mitchwh24**
---
title: "**Can i Buy or switch to one of the old plans without support?**"
description: "We will no longer offer old pricing tiers once these changes take effect on August 15, 2017. Existing customers may stay on their current tier at the current price without the additional support options or they can move to a new sending tier."
---
We will no longer offer old pricing tiers once these changes take effect on August 15, 2017. Existing customers may stay on their current tier at the current price without the additional support options or they can move to a new sending tier.

### CODE REVIEW

File: articles/faq/can-i-switch-to-old-plan [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**Can I buy or switch account to one of the old plans that don’t include support?**
```
Comment (richleland) [inline articles/faq/can-i-switch-to-old-plan]: Per discussion, add metadata. See the [example file](https://raw.githubusercontent.com/SparkPost/support-docs/master/example/article.md) for reference.

**Comment by mitchwh24** [comment]
closing will adjust and try again

**Comment by mitchwh24** [comment]
Added headers - please confirm if this is sane or not

**Comment by richleland** [comment]
Hey @mitchwh24 those need to be in the markdown file, not the pull request description - lmk once you've added that and I'll re-review!

**Comment by richleland** [comment]
Forks don't build due to encrypted travis creds, merging.

**Pull request merged**